src/components/todo/TodoList.jsx

```diff
@@ -1,173 +1,164 @@
-import React, { useState, useEffect } from 'react'
-import axios from 'axios'
-import styled from 'styled-components'
-import { Checkbox } from '@mantine/core';
-import { CloseButton } from '@mantine/core';
-import { List } from '@mantine/core';
-import { Popover, Button, TextInput } from '@mantine/core';
-
-const Task_ul  =styled.ul`
-
-`;
-
-const Task_li  =styled.li`
-list-style:none;
-font-size:25px;
-display:flex;
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import styled from "styled-components";
+import { Checkbox } from "@mantine/core";
+import { CloseButton } from "@mantine/core";
+import { List } from "@mantine/core";
+import { Popover, Button, TextInput } from "@mantine/core";
+
+const TaskList = styled.li`
+  list-style: none;
+  font-size: 25px;
+  display: flex;
 `;
 
-const TaskName_div = styled.div`
-  width:350px;
+const TaskNameDiv = styled.div`
+  width: 350px;
 `;
 
 const TaskName = styled.p`
- width:100%;
- word-wrap: break-word;
+  width: 100%;
+  word-wrap: break-word;
 `;
 
-
-const Button_li = styled.li`
-  list-style:none;
-  padding-top:20px;
- 
+const ButtonList = styled.li`
+  list-style: none;
+  padding-top: 20px;
 `;
 
-const Checkbox_div = styled.div`
-  padding-left:30px;
-  padding-top:13px;
+const CheckboxDiv = styled.div`
+  padding-left: 30px;
+  padding-top: 13px;
 `;
 
-const Edit_div=styled.div`
-`;
-
-const CloseButton_div=styled.div`
-`;
-
-
 function TodoList(props) {
- 
-  const temptodos = {
-    ...props.todo,
-    who: "",
-  };
-
-  const [todos, setTodos] = useState([]);  
-  const [userName, setUserName] = useState([]);  
+  const [todos, setTodos] = useState([]);
   const [todoName, setTodoName] = useState("");
 
   const getTodos = () => {
-     axios
-      .get(process.env.REACT_APP_HOST+"/todos/index")
+    axios
+      .get(process.env.REACT_APP_HOST + "/todos/index")
       .then((res) => {
-        if (res !== ''){
+        if (res !== "") {
           setTodos(res.data);
         }
       })
-      .catch(() => console.error);
-
-  };
-
-  const getTodosUser = (todo,index) => {
-    const user_id = todo.user_id;
-     axios
-      .get(process.env.REACT_APP_HOST+"/user_get/"+user_id)
-      .then((res) => {
-        if (res !== ''){
-          setUserName(res.data.email)
-        }
-        return <>{userName}</>;
-      })
-      .catch(() => console.error);
+      .catch();
   };
 
   const toggleComplete = async (id, index) => {
     const complete = todos[index].complete;
-    await axios.put(process.env.REACT_APP_HOST+"/todos/"+id, {
+    await axios.put(process.env.REACT_APP_HOST + "/todos/" + id, {
       complete: !complete,
     });
     getTodos();
   };
 
-  const editTaskName = async (e,id) => {
-    await axios.put(process.env.REACT_APP_HOST+"/todos/"+id, {
-      name: todoName
+  const editTaskName = async (e, id) => {
+    await axios.put(process.env.REACT_APP_HOST + "/todos/" + id, {
```

Comment:
（want）変数展開で書いたほうがわかりやすいですね！
```suggestion
    await axios.put(`${process.env.REACT_APP_HOST}/todos/${id}`, {
```